src/datachain/lib/file.py

```diff
@@ -964,6 +964,35 @@ def get_fragments(
             yield self.get_fragment(start, min(start + duration, end))
             start += duration
 
+    def save(  # type: ignore[override]
```

Comment:
just to double check - I wonder if need some kind of `@overload` here (I don't have much experience with it though)